test/test-threadpool-cancel.c

```diff
@@ -55,29 +56,27 @@ static void done_cb(uv_work_t* req, int status) {
 }
 
 
-static void saturate_threadpool(void) {
+void saturate_threadpool(unsigned start) {
   uv_loop_t* loop;
-  char buf[64];
   size_t i;
 
-  snprintf(buf,
-           sizeof(buf),
-           "UV_THREADPOOL_SIZE=%lu",
-           (unsigned long)ARRAY_SIZE(pause_reqs));
-  putenv(buf);
+  if (start) {
```

Comment:
Minor thing but can you write this as `start > 0` to make it clear it's an integral and not a boolean?